content/libraries.md

```diff
@@ -114,6 +114,40 @@ Some great folks have written their own libraries to help interact with Datadog.
 
   * [datadog-scala][37] - a Scala API client, written by [Cory Watson][38].
 
+### Community Tracing Libraries
+
+#### C\#
+{: #community-tracing-csharp}
+
+  * [DatadogSharp][89] - A C# Datadog client that supports DogStatsD and APM.
+
+#### Elixir
+{: #community-tracing-elixir}
+
+  * [spandex][90] - A Datadog APM reporting library.
+
+#### Java
+{: #community-tracing-java}
+
+  * [apm-client][91] - A Java client for Datadog APM.
+
+#### OpenTracing
+{: #community-tracing-opentracing}
+
+  * [datadog-tracer-js][92] - OpenTracing tracer implementation for Datadog in JavaScript.
+
+  * [dd-go-opentracing][93] - OpenTracing tracer implementation for Datadog in Go.
+
+#### PHP
+{: #community-tracing-php}
+
+  * [dd-trace-php][94] - A PHP tracing implementation for Datadog.
+
+#### Zipkin
+{: #community-tracing-zipkin}
+
+  * [dd-zipkin-proxy][95] - A simple Zipkin to datadog proxy for Datadog's new tracing feature.
```

Comment:
I realize this description comes directly from the repo, but:

* s/Zipkin to datadog/Zipkin-to-Datadog
* we shouldn't refer to tracing as "new"; we aren't doing that anywhere else

I think just "A simple Zipkin-to-Datadog proxy" is good.